src/create-element.js

```diff
@@ -75,11 +75,14 @@ export function createVNode(type, props, key, ref, original) {
 		// a _nextDom that has been set to `null`
 		_nextDom: undefined,
 		_component: null,
-		constructor: undefined,
-		_original: original
+		constructor: undefined
 	};
 
-	if (original == null) vnode._original = vnode;
+	Object.defineProperty(vnode, '_original', {
```

Comment:
This likely bypasses mangle.json config.

One way I thought of for getting around that was to use `Object.defineProperties`:

```js
Object.defineProperties(vnode, {
  _original: {
    value: original == null ? vnode : original
  }
});
```

Reply:
From the compiled code:

```
return Object.defineProperty(o,"__v",{value:r||o,enumerable:!1,writable:!0})
```